crates/cac_client/src/lib.rs

```diff
@@ -183,42 +169,35 @@ impl Client {
         self.last_modified.read().await.clone()
     }
 
-    pub async fn eval(
+    pub async fn get_resolved_config(
```

Comment:
INFO: Same change-log as earlier, just removed eval function post [this discussion](https://github.com/juspay/superposition/pull/264/files#r1806362850)